static/app/components/performance/vitalsAlert/constants.tsx

```diff
@@ -1,15 +1,17 @@
 import {VitalsKey} from './types';
 
+// these are industry standards determined by Google (https://web.dev/defining-core-web-vitals-thresholds/)
 export const INDUSTRY_STANDARDS: Record<VitalsKey, number> = {
   LCP: 2500,
   FCP: 1800,
   appStartCold: 5000,
   appStartWarm: 2000,
 };
 
+// these were determined using a Looker query and might change over time
 export const SENTRY_CUSTOMERS: Record<VitalsKey, number> = {
   LCP: 948,
   FCP: 760,
-  appStartCold: 4000, // TODO: Update
-  appStartWarm: 1500, // TODO: Update
+  appStartCold: 2260,
+  appStartWarm: 1900,
```

Comment:
can you.. put these in two different PRs lol...

these two things are totally unrelated

Reply:
@evanpurkhiser ugh you’re so annoying

Reply:
Storybook doesn’t even work so I’ll just remove that part

Reply:
see there's a good reason right there 😏

Reply:
@evanpurkhiser lol I hate it even more when you're kind of right, removed the storybook file